FedComm/src/FedComm.cpp

```diff
@@ -149,41 +160,42 @@ void GetFrameworkInfoFromGossiper(int gossiperSockfd)
     }
     else if (MsgType == MSG_TYPE_HEARTBEAT)
     {
-       numChar = write(gossiperSockfd,(void *) &ack,1);
-       LOG(INFO) << "FEDERATION: Sent Heart Beat to Gossiper";
+      numChar = write(gossiperSockfd,(void *) &ack,1);
+      LOG(INFO) << "FEDERATION: Sent Heart Beat to Gossiper";
     }
     else if (MsgType == MSG_TYPE_DATA) // If data is sent by the Gossiper
     {
        // Read the length of payload
-       numChar = read(gossiperSockfd, &buf, 4);
```

Comment:
I feel all the return values should be NULL checked, the variables declared are unsigned int so those cannot go in negative, am i right in this ?